### ISSUE TITLE
Bundler connection to rubygems.org failed; Ruby & RubyGems connection succeeded

### ISSUE THREAD

**Issue by EvanGrillo** [rubygems/rubygems#3226]
## input
```
ruby -ropen-uri -e 'eval open("https://git.io/vQhWq").read'
```
## output
```
Here's your Ruby and OpenSSL environment:

Ruby:           2.5.1p57 (2018-03-29 revision 63029) [x86_64-darwin16]
RubyGems:       2.7.7
Bundler:        1.16.2
Compiled with:  OpenSSL 1.0.2p  14 Aug 2018
Loaded version: OpenSSL 1.0.2p  14 Aug 2018
SSL_CERT_FILE:  /usr/local/etc/openssl/cert.pem
SSL_CERT_DIR:   /usr/local/etc/openssl/certs

With that out of the way, let's see if you can connect to rubygems.org...

Bundler connection to rubygems.org:       failed  ❌  (execution expired)
RubyGems connection to rubygems.org:      success ✅
Ruby net/http connection to rubygems.org: success ✅

Although your Ruby installation and RubyGems can both connect to rubygems.org, Bundler is having trouble. The most likely way to fix this is to upgrade Bundler by running `gem install bundler`. Run this script again after doing that to make sure everything is all set. If you're still having trouble, check out the troubleshooting guide at http://ruby.to/ssl-check-failed 📦
(eval):136: warning: constant OpenSSL::SSL::SSLContext::METHODS is deprecated
```

**Comment by deivid-rodriguez** [comment]
Anyone who participated in this thread is using ipv6? You can check https://ipv6-test.com/, for example. I'm doing some triaging for issues that could get solved by #2662.

**Comment by deivid-rodriguez** [comment]
Nobody answered here, so I'll assume this is no longer an issue.
